### PR TITLE
Card layout

### LINKED ISSUE #9: Once a filter is selected, can't un-select through UI
On the homepage, after selecting a filter, there is no option to un-select filters to return to "all announcements."

### LINKED ISSUE #10: Responsive styles - navigation
- Collapsed nav styles need to be updated to match the latest main site styles
- Links should be white on hover, not gold
- Collapse button is not positioned correctly and overlaps/is not accessible in some cases
screen shot 2017-09-28 at 9 22 02 am
- Font size should be reduced slightly at -lg breakpoint so that all links fit within a single line (see main site styles)
screen shot 2017-09-28 at 9 22 48 am

### LINKED ISSUE #11: Responsive styles - frontend announcement list
- Main site navigation needs some extra breathing room between it and the Announcements h1.  We've got main site v3 set up to use a dark gray navbar on pages without a header image--will post a link in Slack
- Consider breaking out of the masonry layout at the -md breakpoint and below for announcement card contents to not clip out of their containers
screen shot 2017-09-28 at 9 24 30 am
- Consider using a smaller font size for the Ongoing Announcements heading; make sure it does not cause horizontal scrollbars at smaller screen sizes
- Consider removing the "Learn More" buttons on each announcement card with a link wrapped around each announcement title
- Consider using a smaller font size for announcement titles, and possibly for audience and contact info

### PR DESCRIPTION
**Pull request by jmbarne3**
Bug Fixers:
* Fixes #10 by implementing Main-Site-Theme v3.0.0 styles for navigation.
* Fixes #11 by updating col widths at various breakpoints and adding the `h-100` class to the columns.

Enhancements:
* Fixes #9 by allowing filters to be toggleable.

### CODE REVIEW

**Comment by jmbarne3** [comment]
Also fixes #7

File: templates/home.html [hunk 1]
```diff
@@ -20,12 +24,12 @@
...
 <div class="announcements">
   <div class="row">
-    <div class="col-md-9">
+    <div class="col-lg-9">
       {% if announcements %}
       <div class="row">
         {% for item in announcements %}
-        <div class="col-6">
-          <div class="card mb-4">
+        <div class="col-12 col-sm-12 col-md-12 col-lg-6 mb-4">
```
Comment (cjg89) [inline templates/home.html]: .col-12, .col-sm-12 and .col-md-12 classes shouldn't be needed here

File: templates/home.html [hunk 2]
```diff
@@ -59,7 +63,7 @@ <h2 class="h3 card-header">{{ item.title }}</h2>
       <p>No announcements found.</p>
       {% endif %}
     </div>
-    <div class="col-md-3">
+    <div class="col-lg-3">
       <h2>Ongoing Announcements</h2>
```
Comment (cjg89) [inline templates/home.html]: Consider adding a .h4 class to this heading to avoid clipping

File: templates/partials/_header.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-        <div id="header-menu" class="collapse navbar-collapse">
-            <div class="container">
-                <ul class="nav navbar-nav nav-fill w-100">
+    <nav class="navbar navbar-toggleable-md navbar-custom navbar-inverse py-2 py-sm-4 bg-default" id="header-nav-wrap" role="navigation">
+        <div class="container">
+            <button class="navbar-toggler ml-auto mr-0 collapsed" type="button" data-toggle="collapse" data-target="#header-menu" aria-controls="header-menu" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-text">Navigation</span>
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div id="header-menu" class="collapse navbar-collapse" area-expanded="false">
```
Comment (cjg89) [inline templates/partials/_header.html]: Typo `area-expanded` should be `aria-expanded`

**Pull request merged**